src/components/invite/useInviteLevel.js

```diff
@@ -0,0 +1,15 @@
+import { useState } from 'react'
+import GDStore from '../../lib/undux/GDStore'
+
+const useInviteLevel = () => {
+  const gdstore = GDStore.useStore()
```

Comment:
1. loading from contacts should be here
2. there's an useStoreProp hook exported from GDStore allows you to do const [inviteLevel, setInviteLevel] = useStoreProp('inviteLevel'_
3. hook should return only level value
4. hook should be called from dashboard